src/OrchardCore.Modules/OrchardCore.Admin/AdminAreaControllerRouteMapper.cs

```diff
@@ -19,7 +19,10 @@ public AdminAreaControllerRouteMapper(IOptions<AdminOptions> adminOptions)
 
         public bool TryMapAreaControllerRoute(IEndpointRouteBuilder routes, ControllerActionDescriptor descriptor)
         {
-            if (descriptor.ControllerName == "Admin" || descriptor.ControllerTypeInfo.GetCustomAttributes<AdminAttribute>().Any())
+            if (descriptor.ControllerName == "Admin" ||
+                descriptor.ControllerTypeInfo.GetCustomAttribute<AdminAttribute>() != null ||
+                descriptor.MethodInfo.GetCustomAttribute<AdminAttribute>() != null
+                )
```

Comment:
Cool, so in case of an admin action on a non admin controller, i forgot this one ;)